Readme.md

```diff
@@ -511,6 +512,49 @@ completed
 ```
 This example emits 2, 5 (`NSDecimalNumber` Type).
 
+#### weak
+
+Convenience versions of `subscribe` and `flatMap` that provide a safe unretained reference to a specific object as an argument of their respective closures.
+
+```swift
+private class Tool {
+    
+    deinit {
+        print("deinit")
+    }
+    
+    func display(_ value: Any) {
+        print(value)
+    }
+    
+    func produce(_ value: Int) -> Observable<Int> {
+        return Observable.of(value * 2, value * 3)
+    }
+}
+
+let observable = Observable.of(2, 5, 7)
+
+var tool: Tool? = Tool()
+    
+observable.flatMap(weak: tool!) { (strongTool, value) in
```

Comment:
We shouldn't use force unwrap in examples... the examples should provide relatively clean standards of usage so they don't confuse a developer.

In general might be nice to put some of the examples you put in the PR itself instead. They seemed to be much more descriptive than these and more "real-life".

Reply:
done